spec/models/ext_management_system_spec.rb

```diff
@@ -18,12 +18,13 @@
 
   let(:all_types_and_descriptions) do
     {
-      "ansible_tower_configuration" => "Ansible Tower Configuration",
+      "ansible_tower_automation"    => "Ansible Tower Automation",
       "azure"                       => "Azure",
       "azure_network"               => "Azure Network",
       "ec2"                         => "Amazon EC2",
       "ec2_network"                 => "Amazon EC2 Network",
       "ec2_ebs_storage"             => "Amazon EBS",
+      "embedded_ansible_automation" => "Embedded Ansible Automation",
```

Comment:
`embedded_ansible` seems fine to me.  We should change it to that in a follow up PR.

It's a bit like `ec2`, where we have newer managers that were added later for network and ebs (which really should be "storage").  We could start with a baseline `embedded_ansible`, then tack on new managers if they ever pop up.